cfgmgr/vxlanmgr.h

```diff
@@ -0,0 +1,68 @@
+#ifndef __VLANMGR__
+#define __VLANMGR__
+
+#include "dbconnector.h"
+#include "producerstatetable.h"
+#include "orch.h"
+
+#include <map>
+#include <memory>
+#include <string>
+
+namespace swss {
+
+class VxlanMgr : public Orch
+{
+public:
+    VxlanMgr(DBConnector *cfgDb, DBConnector *appDb, DBConnector *stateDb, const vector<std::string> &tableNames);
+    using Orch::doTask;
+
+    typedef std::map<std::string, std::string> VxlanInfo;
```

Comment:
From the usage of this, It will be more readable and clearer to define it as a struct. 

```
struct {

string vxlan_name;

string vni;

string source_ip;
}
```